tests/samples/expected/ocaml.ml

```diff
@@ -471,6 +471,7 @@ module type Printer = sig
     element printer and [sep] as separator between elements. *)
   val print_list : string -> 'a printer -> 'a list printer
   val print_name : name printer
+
```

Comment:
This blank line appears in the `input` file, but was deleted in the previous version